### PR TITLE
Moved Tip to Ouroboros.Network.Block

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: ouroboros-network/src/Ouroboros/Network/Block.hs [hunk 1]
```diff
@@ -186,6 +190,33 @@ castPoint (Point (At (Point.Block slot hash))) = Point (block slot hash)
 blockPoint :: HasHeader block => block -> Point block
 blockPoint b = Point (block (blockSlot b) (blockHash b))
 
+{-------------------------------------------------------------------------------
+  Tip of a chain
+-------------------------------------------------------------------------------}
+
+-- | Used in chain-sync protoco to advertise the tip of the server's chain.
```
Comment (mrBliss) [inline ouroboros-network/src/Ouroboros/Network/Block.hs]: missing l after protoco

**Comment by mrBliss** [comment]
Oh, this type should be used by `Ouroboros.Network.Protocol.ChainSync.Examples*`

**Comment by coot** [comment]
Yes it could, good point, also in some of the test we use a pair instead of a `Tip`; I'll leave it as issue.

**Comment by coot** [comment]
Ref #1072

**Pull request merged**